pulsar-broker/src/main/java/org/apache/pulsar/common/naming/TopicBundleAssignmentStrategy.java

```diff
@@ -23,5 +23,7 @@
 public interface TopicBundleAssignmentStrategy {
     NamespaceBundle findBundle(TopicName topicName,  NamespaceBundles namespaceBundles);
 
+    long getHashCode(String name);
```

Comment:
It's better to make the name of this method more specific. It's better to use the type `TopicName` as well and provide a default implementation. Please also add javadoc with proper description of the method.

```suggestion
    default long calculateBundleHashCode(TopicName topicName) {
        return Hashing.crc32().hashString(topicName.toString(), StandardCharsets.UTF_8).padToLong();    
    }
```

Reply:
Change as suggested, thank you. @lhotari